charts/piraeus/charts/etcd/values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for etcd.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+peerPort: 2380
+clientPort: 2379
+component: "etcd"
+replicas: 3
+image:
+  repository: "k8s.gcr.io/etcd-amd64"
+  tag: "3.2.26"
```

Comment:
This is quite an old release (11 Jan 2019). Have you looked for newer images?

Reply:
I only did a very brief search for newer tags on gcr.io, but have not found anything. I will try https://quay.io/repository/coreos/etcd

Reply:
That's an improvement. Anything "coreos" related might be abandoned or moved since the project in its previous form has been stopped. We could even continue to use the bitnami images, since the issues were in the chart.

Reply:
Actually, it seems the etcd team uses [two registries](https://github.com/etcd-io/etcd/releases/tag/v3.4.9):
* primary: https://gcr.io/etcd-development/etcd
* secondary: https://quay.io/coreos/etcd
So it looks like the coreos image won't go away anytime soon. I will still switch to the primary source

Reply:
Changed to `gcr.io/etcd-development/etcd:v3.4.9`

Reply:
from my experience the updated image seems to work